docs/release.rst

```diff
@@ -0,0 +1,17 @@
+.. toctree::
+   :maxdepth: 1
+
+Release
+=======
+This section is for package maintainers.
+
+Release Maven Jar
+-----------------
+We release the rikai jar to the maven artifactory using
+`sbt-sonatype <https://github.com/xerial/sbt-sonatype>`_ in conjunction with
+`sbt-release <https://github.com/sbt/sbt-release>`_. You can follow `these instructions
+<https://www.scala-sbt.org/1.x/docs/Using-Sonatype.html>`_ to setup :code:`sbt-sonatype`.
+
+Sanpshot builds can be published to Sonatype snapshots repo using :code:`sbt publishSigned`.
+Release builds can be published to Sonatype staging repo then sync'd to maven using
+:code:`sbt release`
```

Comment:
Curious what do we need to do for automatically release in github actions?
Or are we planning to do manual release for the timebeing?

Reply:
i didn't find an official integration but some of the actions in the marketplace looked promising